### DISCUSSION TITLE
CKAN POSE Project publishes CKAN Instance Analysis

### DISCUSSION THREAD

**Discussion by jqnatividad** [ckan/ckan#7670] [CKAN 3.0]
Folks,
The POSE Project team just published an updated survey of CKAN instances as part of its CKAN Ecosystem Discovery.

https://civicdataecosystem.org/2023/06/19/analysis.html

Your feedback is most welcome (as we mounted a [giscus](https://giscus.app/) integration to allow Github-backed comments)!

cc @rgradeck @noramattern @rossreillypitt @saylorsd @lizmo @drw @samibaig

**Comment by thegostev** [comment]
Enjoyed reading the report, it was an insightful read, great job @jqnatividad and @rossreillypitt. I think it's an important job to do it as a regular audit in future. Possibly by developing an efficient approach to cataloging CKAN instances for higher representativeness. A few insights to mention: 

1. CKAN can efficiently handle more than a 1,000,000 of datasets
3. CKAN is lagging with adoption of new versions: https://github.com/ckan/ckan/issues/7663
4. Without a call-home mechanism implemented, we're retrospective and not up to date with knowing the amount of active CKAN instances around the globe: https://github.com/ckan/ckan/issues/7587
